pom.xml

```diff
@@ -9,7 +9,7 @@
 
 	<groupId>com.apigee.edge.config</groupId>
 	<artifactId>apigee-config-maven-plugin</artifactId>
-	<version>1.3.6-SNAPSHOT</version>
```

Comment:
@Abhishek-chouksey - pls do not change the version. The current version is 1.3.5, so the pom.xml will point to 1.3.6-SNAPSHOT. During the release, it will get pushed as 1.3.6 and this file will automatically be updated to 1.3.7-SNAPSHOT. So please revert this change